hazelcast/proxy/map.py

```diff
@@ -167,43 +371,128 @@ def get_all(self, keys):
         return self._get_all_internal(partition_to_keys)
 
     def get_entry_view(self, key):
+        """
+        Returns the EntryView for the specified key.
+
+        **Warning:
+        This method returns a clone of original mapping, modifying the returned value does not change the actual value
+        in the map. One should put modified value back to make changes visible to all nodes.**
+
+        **Warning 2:
+        This method uses GetHashCode and Equals of binary form of the key, not the actual implementations of GetHashCode
+        and Equals defined in key's class.**
+
+        :param key: (Object), the key of the entry.
+        :return: (EntryView), EntryView of the specified key.
+
+        See :ref:`EntryView` for more info about EntryView.
+        """
         check_not_none(key, "key can't be None")
         key_data = self._to_data(key)
         return self._encode_invoke_on_key(map_get_entry_view_codec, key_data, key=key_data, thread_id=thread_id())
 
     def is_empty(self):
+        """
+        Returns true if this map contains no key-value mappings.
+
+        :return: (Bool), true if this map contains no key-value mappings.
+        """
         return self._encode_invoke(map_is_empty_codec)
 
     def is_locked(self, key):
+        """
+        Checks the lock for the specified key. If the lock is acquired then returns true, else returns false.
+
+        **Warning:
+        This method uses GetHashCode and Equals of binary form of the key, not the actual implementations of GetHashCode
+        and Equals defined in key's class.**
+
+        :param key: (Object), the key that is checked for lock
+        :return: (Bool), true if lock is acquired, false otherwise.
+        """
         check_not_none(key, "key can't be None")
         key_data = self._to_data(key)
         return self._encode_invoke_on_key(map_is_locked_codec, key_data, key=key_data)
 
     def key_set(self, predicate=None):
+        """
+        Returns a List clone of the keys contained in this map or the keys of the entries filtered with the predicate if
+        provided.
+
+        **Warning:
+        The list is NOT backed by the map, so changes to the map are NOT reflected in the list, and vice-versa.**
+
+        :param predicate: (Predicate), predicate to filter the entries (optional).
+        :return: (List), a list of the clone of the keys.
+
+        See :ref:`Predicate` for more info about predicates.
+        """
         if predicate:
             predicate_data = self._to_data(predicate)
             return self._encode_invoke(map_key_set_with_predicate_codec, predicate=predicate_data)
         else:
             return self._encode_invoke(map_key_set_codec)
 
     def load_all(self, keys=None, replace_existing_values=True):
+        """
+        Loads all keys into the server side store or load the given keys if provided.
+
```

Comment:
Loads all keys from the store at server side or loads the given keys if provided.